libdnf/dnf-repo.cpp

```diff
@@ -1832,9 +1832,9 @@ dnf_repo_update(DnfRepo *repo,
 
     /* see dnf_repo_check_internal */
     if (!dnf_context_get_enable_filelists(priv->context)) {
-        const gchar *blacklist[] = { "filelists", NULL };
+        const gchar *yumblist[] = { "filelists", NULL };
```

Comment:
I think this is more commonly changed to e.g. "skiplist" or "omit_list".

Reply:
Thanks Colin. Tbh. I don't think it matters for this case, I just named the local variable after the actual option, I think "blist" is fine. It could also stand for "blocklist", one of the suggested replacement names. For that reason I didn't want to pick a random different word, here we can wait for what new name (hopefully) becomes the industry standard.

Reply:
yum typically uses `exclude`/`include`, so eventually we should refit around this.

Reply:
How about `exclude(d)_metadata_types`?
Or something similar that explicitly indicates what is excluded.

Reply:
@dmach changed.

Reply:
Thanks! Merging.